stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -371,6 +367,7 @@ async def patch_item(
         collection_id: str,
         item_id: str,
         patch: Union[PartialItem, List[PatchOperation]],
+        content_type: Optional[str],
```

Comment:
I believe this should be `content_type: Optional[str] = "application/json"

because earlier you had `content_type = kwargs["request"].headers.get("content-type", "application/json")`

Reply:
or you can add a line 

```python
content_type = content_type or "application/json"
```

Reply:
maybe we could have literal 
```
Literal["application/json-patch+json", "application/merge-patch+json", "application/json"]
```

just to `advertise` what are the supported content type